components/pd_client/src/config.rs

```diff
@@ -13,7 +13,7 @@ use tikv_util::config::ReadableDuration;
 pub struct Config {
     /// The PD endpoints for the client.
     ///
-    /// Default is empty.
+    /// Default is `"127.0.0.1:2379"`.
```

Comment:
Just updated the comment because `Config::Default()` sets it to `127.0.0.1:2379`. See also https://github.com/tikv/tikv/blob/3ef232e7f2f4ad8087c8268026cefa958b637194/components/pd_client/src/config.rs#L44